tchannel/schemes/json.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import json
+
+from tornado import gen
+
+from . import JSON
+
+
+class JsonArgScheme(object):
+    """Semantic params and serialization for json."""
+
+    NAME = JSON
+
+    def __init__(self, tchannel):
+        self._tchannel = tchannel
+
+    @gen.coroutine
+    def __call__(self, service, endpoint, body=None,
+                 headers=None, timeout=None):
```

Comment:
I remember there is an issue with using kwargs. It will be unclear for caller to know those arguments.